internal/command/user_test.go

```diff
@@ -920,7 +921,7 @@ func TestCommandSide_RemoveUser(t *testing.T) {
 			ctx                    context.Context
 			orgID                  string
 			userID                 string
-			cascadeUserMemberships []*domain.UserMembership
+			cascadeUserMemberships []*query.Membership
```

Comment:
this test might have been changed accidentally...

the actual function still uses the domain and not the query package
and I'm not sure if using the query pkg in the command is meaningful